docs-site/content/guide/high-availability.md

```diff
@@ -272,3 +272,137 @@ final config = Configuration(
 
   </template>
 </Tabs>
+ 
+ ## Deploy a Typesense cluster to Docker Swarm  
```

Comment:
Could you remove this section?

Reply:
done

Reply:
i will change reference to `docker` nodes

Reply:
@jasonbosco Please review.